src/app/versioning/version.service.ts

```diff
@@ -138,7 +133,7 @@ export class VersionService {
 
     private connected() {
         let connection = null;
-        this._connectSvc.active.first().subscribe(c => {
+        this._connectSvc.active.pipe(first()).subscribe(c => {
```

Comment:
This looks like a legacy code. For safety, please check if we need to call unsubscribe or not.